tools/devtool

```diff
@@ -743,9 +743,9 @@ cmd_test() {
       # It seems that even if the tests using huge pages run sequentially on ag=1 agents, right-sizing the huge pages
       # pool to the total number of huge pages used across all tests results in spurious failures with pool depletion
       # anyway (something else on the host seems to be stealing our huge pages, and we cannot "ear mark" them for
-      # Firecracker processes). Thus, just allocate 8GB of them and call it a day.
+      # Firecracker processes). Thus, just allocate 48GB of them and call it a day.
       say "Setting up huge pages pool"
-      num_hugetlbfs_pages=4096
+      num_hugetlbfs_pages=24552
```

Comment:
Were you able to somehow validate that this solves the problem?

Reply:
Yes, I ran the tests in my pipeline and they no longer fail.